### PR TITLE
[DOCS] Schema change detection 

### PR DESCRIPTION
**Pull request by klavavej**
Resolves https://greatexpectations.atlassian.net/browse/DOC-989 according to the plan linked in that issue.

Note that the base branch for this PR is temporarily set to the SUJ-E docs branch. This PR has a dependency on the SUJ-E docs updates. After SUJ-E merges, I'll update the base here to `develop` 

- [X] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [X] Appropriate tests and docs have been updated

For more information about contributing, visit our [community resources](https://docs.greatexpectations.io/docs/core/introduction/community_resources#contribute-code-or-documentation).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md [hunk 1]
```diff
@@ -173,7 +174,7 @@ Gallery](https://greatexpectations.io/expectations/expect_table_column_count_to_
 
 ## Examples
 
-**GX Cloud** provides a visual interface to create and run schema validation workflows. The GX Cloud workflow for validating data schema is to create a Data Asset, define Expectations, run a Validation, and review Validation Results.
+**GX Cloud** provides a visual interface to create and run schema validation workflows. The GX Cloud workflow for validating data schema is to create Data Assets with automatically generated Expectations, manually define additional Expectations, run Validations, and review Validation Results.
```
Comment (shiplet) [inline docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md]: Should `validating data schema` be `validating a data schema`?

**Review by shiplet — APPROVED** [review]
Looks good to me! Only one non-blocking question

**Review by annabarr — APPROVED** [review]
this lgtm!

File: docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md (hunk 1 above)
Comment (klavavej) [reply docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md]: good question! I'm going to change this to "validating data schemas" (plural) so I can still say "automatically generated Expectations" (plural) to future proof this a bit so it works for the current state of there only being one Expectation that can be autogenerated and will continue to work after we add more.

Comment (klavavej) [reply docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md]:
```suggestion
**GX Cloud** provides a visual interface to create and run schema validation workflows. The GX Cloud workflow for validating data schemas is to create Data Assets with automatically generated Expectations, manually define additional Expectations, run Validations, and review Validation Results.
```

**Pull request merged**